apps/src/code-studio/components/stageExtras/MazeThumbnail.jsx

```diff
@@ -5,7 +5,7 @@ import {getSubtypeForSkin} from '@cdo/apps/maze/mazeUtils';
 import MazeMap from '@cdo/apps/maze/mazeMap';
 import drawMap from '@cdo/apps/maze/drawMap';
 
-const assetUrl = path => '/blockly/' + path;
+const assetUrl = path => 'https://studio.code.org/blockly/' + path;
```

Comment:
:expressionless: I feel like your PR description misrepresents this change. Might be fine though.  Where do we use the `MazeThumbnail` component - only on levelbuilder?

I wonder if we can do an inline replace in the storybook build, or pass an assetUrl function into this component, or something that will only do this for storybook and/or will work for all of storybook, not just this component.